aten/src/ATen/gen.py

```diff
@@ -180,7 +180,7 @@ def check_all_files_written(self):
 
 # scalar_name, c_type, accreal, th_scalar_type, is_floating_type
 scalar_types = [
-    ('Bool', 'uint8_t', 'BoolAccrealNotDefined', 'uint8_t', False),
+    ('Bool', 'bool', 'BoolAccrealNotDefined', 'uint8_t', False),
```

Comment:
Huh, I wonder why we didn't apply this change earlier

Reply:
interestingly, `th_scalar_type` is also wrong, but it's unused.  Can you kill it everywhere in a different PR?

Reply:
Will kill th_scalar_type in a separate PR